doc/io/dms/managed.md

```diff
@@ -0,0 +1,21 @@
+# AWS DMS Managed
+
+## About
+Conduct a data migration from any source supported by AWS DMS into a database
+table on [CrateDB Cloud], exclusively using managed infrastructure components.
+
+:::{note}
+This is a work in progress. Please contact our data engineers to get started.
+:::
+
+## Configuration
+1. Set up a DMS instance to replicate data to an Amazon Kinesis Data Stream.
+2. Take a note about the AWS ARN of that Kinesis Data Stream,
+   for example `arn:aws:kinesis:eu-central-1:831394476016:stream/testdrive`.
+3. Reach out to CrateDB support, to make CrateDB Cloud connect to your data
+   stream, in order to converge it into your CrateDB Cloud instance.
```

Comment:
This would be the gist of the whole story when aiming to use a "managed only, hands-free" way of doing data migration operations, for now in an "ad hoc deployment mode"?